cupy/linalg/einsum.py

```diff
@@ -340,6 +352,13 @@ def reduced_binary_einsum(arr0, sub0, arr1, sub1, sub_others):
     sub_out = sub_b + sub_l + sub_r
     assert set(sub_out) <= set_others, 'operands should be reduced: unary sum'
 
+    if len(contract_dims) == 0:
```

Comment:
Please write a short comment about this shortcut.

Reply:
I will add the comment.